docs/commands/model.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Awesome - such a great idea to have a tool that can output vital information about each model!